site/src/components/data/financial.md

```diff
@@ -18,11 +23,28 @@ The following example:
 
 Generates the following:
 
-{{{ dynamic-include 'codepen' html="financial-example-html" js="financial-example-js" }}}
-
 {{{financial-example-html}}}
 <script type="text/javascript">
 {{{financial-example-js}}}
-</script>
+</script> 
+
+{{{ dynamic-include 'codepen' html="financial-example-html" js="financial-example-js" }}}
+
+## Stream
+The component exposes a stateful streaming interface.
```

Comment:
I would try and make it clearer what the difference between the stream and the base component is.